assets/styles/components/_wysiwyg.scss

```diff
@@ -0,0 +1,363 @@
+        & + table,
+        & + figure {
+            margin-top: size-clamp('md');
+        }
+
+        & + h1,
+        & + h2,
+        & + h3,
+        & + h4,
+        & + h5,
+        & + h6 {
+            margin-top: size-clamp('md');
+        }
+    }
+
+    h4,
+    h5,
+    h6 {
+        margin-top: size-clamp('lg');
+        margin-bottom: size-clamp('sm');
+
+        & + p,
+        & + ul,
+        & + ol,
+        & + table,
+        & + figure {
+            margin-top: size-clamp('sm');
+        }
+
+        & + h1,
+        & + h2,
+        & + h3,
+        & + h4,
+        & + h5,
+        & + h6 {
+            margin-top: size-clamp('sm');
+        }
+    }
+
+    h4 {
+        // @include heading-h5;
+    }
+
+    h5 {
+        // @include heading-h6;
+    }
+
+    h6 {
+        // @include heading-h6;
+    }
+
+
+    // ==========================================================================
+    // Lists
+    // ==========================================================================
+    ul,
+    ol {
+        position: relative;
+        margin: size-clamp('md') 0;
+
+        >li {
+            padding-left: rem(30px);
+        }
+    }
+
+    ul {
+        & > li {
+            position: relative;
+
+            & + li {
+                margin-top: 0;
+            }
+
+            &::before {
+                content: "";
+                position: absolute;
+                left: 0;
+                top: 9px;
+                width: 8px;
+                height: 8px;
+                background-color: currentColor;
+                border-radius: 100%;
+            }
+        }
+    }
+
+    ol {
+        counter-reset: counter;
+
+        & > li {
+            counter-increment: counter;
+            position: relative;
+
+
+            & + li {
+                margin-top: 0;
+            }
+
+            &::before {
+                position: absolute;
+                left: 0;
+                content: counter(counter);
+            }
+        }
+    }
+
+    // ==========================================================================
+    // Link
+    // ==========================================================================
+    a {
+        display: inline;
+        text-decoration: underline;
+        text-decoration-thickness: 1px;
+        color: currentColor;
+        word-wrap: break-word;
+
+        &:hover {
+            text-decoration-thickness: 2px;
+        }
+    }
+
+    // ==========================================================================
+    // Image
+    // ==========================================================================
+    img {
+        width: 100%;
+        margin: size-clamp('lg') 0;
+    }
+
+    // ==========================================================================
+    // iFrame
+    // ==========================================================================
+    iframe {
+        width: 100%;
+        height: auto;
+        aspect-ratio: 16 / 9;
```

Comment:
> `aspect-ratio: 16 / 9;`

This is fine for the average video-based embed but might not work well for more vertical-based integrations.

Reply:
Sure, I assumed we mainly add video formatted iframes into wysiwyg and that could be overwritten if needed. But it's removed, so there will be no ratio by default.